dgbpy/hdf5.py

```diff
@@ -182,6 +183,48 @@ def isMultiLabelRegression( info ):
     return False
   return getNrOutputs( info ) > 1
 
+def rm_tree(pth):
+    pth = Path(pth)
+    for child in pth.glob('*'):
+        if child.is_file():
+            child.unlink()
+        else:
+            try:
+              rm_tree(child)
+            except OSError:
+              pass
+    try:
+      pth.rmdir()
+    except OSError:
+      pass
+
+def getLogDir( withtensorboard, examplenm, basedir, clearlogs, args ):
```

Comment:
here you may add the platform as part of the dir name, in case the same example file is used for Pytorch and tensorflow training, or are they compatible, i.e. can a same board show the logs from both platforms?